mlx ui pod: "EACCES: permission denied, mkdir '/workspace/build'"

**Describe the bug**

Trying the MLX install on my openshift cluster this am, so the steps are pretty easy
```
1. Have an existing cluster and be logged in, etc.
2. make sure kustomize is 3.2.0, looks good
3. clone the manifests and switch to the mlx-single-user branch
4. Run:
while ! kustomize build example | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
```
I suspect my cluster needs more configuration though, after step 1, because I've got what looks like pvc permission issues, for example:
```
oc get pods -n kubeflow |grep ui
ml-pipeline-ui-9d7d4855d-9m78b                    1/1     Running             0          18m
mlx-ui-c47bd9f9c-qgvf6                            0/1     CrashLoopBackOff    7          18m

oc logs -f mlx-ui-c47bd9f9c-qgvf6
> aiops-frontend@0.1.0 build /workspace
> react-scripts build

Failed to compile.

EACCES: permission denied, mkdir '/workspace/build'

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! aiops-frontend@0.1.0 build: `react-scripts build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the aiops-frontend@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```
**To Reproduce**

See above

**Expected behavior**

Was expecting the mlx-ui pod to start successfully.

**Screenshots**

n/a

**Environment**

 - OS: Deploying from a Mac while oc logged into the OpenShift cluster
 - Browser: chrome
 - MLX Version:  If the UI was up, maybe I could see the version...
 - OpenShift Version 4.8.39

**Additional context**

None so far, thank you!

--- a/dashboard/origin-mlx/Dockerfile
+++ b/dashboard/origin-mlx/Dockerfile
@@ -19,4 +19,4 @@ RUN cd server && npm install && npm run build
 
 # start app
 ENV NODE_ENV=production
-CMD ["sh", "-c", "npm run build && mv build/ proc/ && node server/dist/server.js proc/ $PORT"]
+CMD ["sh", "-c", "npm run build && node server/dist/server.js build/ $PORT"]

--- a/dashboard/origin-mlx/src/components/Detail/DatasetDetail.tsx
+++ b/dashboard/origin-mlx/src/components/Detail/DatasetDetail.tsx
@@ -14,7 +14,6 @@ import { getUserInfo, hasRole } from '../../lib/util';
 import RunView from '../RunView'
 import RelatedAssetView from '../RelatedAssetView';
 import MarkdownViewer from '../MarkdownViewer';
-import LoadingMessage from '../LoadingMessage';
 import MetadataView from '../MetadataView';
 
 const isAdmin = hasRole(getUserInfo(), 'admin');

--- a/dashboard/origin-mlx/src/components/Detail/KFServingDetail.tsx
+++ b/dashboard/origin-mlx/src/components/Detail/KFServingDetail.tsx
@@ -191,7 +191,7 @@ export default class KFServingDetail extends React.Component<KFServingDetailProp
     }
     else {
       predictorTimestamp = service.metadata.creationTimestamp
-      predictorStatusIcon = service.status.activeModelState == "Ready" 
+      predictorStatusIcon = service.status.activeModelState === "Ready" 
         ? <CheckCircleIcon className="check-icon"/>
         : <ErrorIcon className="error-icon"/>
     }

